src/js/components/leftPanel/analysisPanel/analysisTranslations.ts

```diff
@@ -0,0 +1,46 @@
+export default {
+  RUN_ANALYSIS_BUTTON_TEXT: {
```

Comment:
@vaidotasp what are your thoughts on using camel case in this file? I noticed we've used camel case in previous content config files. Also, I was thinking we could name the object `runAnalysisButton`?